extide/libs.gradle/external/gradle-tooling-api-7.0-rc-2-license.txt

```diff
@@ -1,7 +1,7 @@
 Name: Gradle Wrapper
 Description: Gradle Tooling API
-Version: 6.7
-Files: gradle-tooling-api-6.7.jar
+Version: 7.0-rc-2
+Files: gradle-tooling-api-7.0-rc-2.jar
```

Comment:
Is it necessary to upgrade to RC2 for the 12.4 release ? Gradle policy document says:
```
 Release candidates are supported for as long as the release window is open, but they are not intended to be used long term for production use.
```
So we could end up with an unsupported (?) RC in our production / release build ?

Reply:
Well, that's a good question, and I'm happy that I'm not wearing the RM hat this time.
It is only necessary to upgrade if we expect the IDE running on jdk16 be able to work with Gradle projects.
We could expect that the final 7.0 Gradle come out during our beta/rc cycles (though there's no guarantee on that)
Or we can add a note to the release notes that this config is not supported.
Also with both options we can add a patch release when the final Gradle 7.0 would be available.

If I'd be the RM, I might risk taking up 7.0-rc-2 and I totally understand if someone would not take that one.